### DISCUSSION TITLE
Question about modding player characters and song variants

### DISCUSSION THREAD

**Discussion by PSI3x** [FunkinCrew/Funkin#3399] [Q&A] [answered]
I’ve been looking through the [slight amount of documentation on modding in player characters](https://funkincrew.github.io/funkin-modding-docs/05-custom-playable-characters/05-01-required-assets.html) and there’s something I’d like to ask: Is a way to specify somewhere, such as the metadata json for the specific song, for a character to use the default instrumental and opponent .ogg files in their version of the song?
If not, is this a possible planned addition? I believe it would be very beneficial to add from a filesize optimization, as I believe it is best to not include arbitrary copies of the same sound files for both the default instrumental and opponent voices.

**Comment by Lasercar** [comment, marked as answer]
If you don't want it to use an alt instrumental, don't set one for the variation. It'll just use the default variation's instrumental instead. 

Same with the voices, if it can't find a voice for the variation, it'll default (ha) to the default variation's voice for that character.

**Reply by PSI3x** [reply]
Very interesting, I assumed it required an accompanying opponent track, but it's very good to hear that it can just use the default. Thank you very much for the answer!
